modules/core/src/v2/pendingApproval.ts

```diff
@@ -354,6 +360,28 @@ export class PendingApproval {
         prebuildParams.hop = true;
       }
 
+      if (!recipients.length) {
+        // no recipients - this is a consolidation transaction
+        const filteredParams = _.pick(params, [
```

Comment:
Why do you have to filter the params? I think this is dangerous since we might forget about this when we add more build params in the future.

Reply:
you're right, no need for this since the params come straight from the pending approval object.